examples/apps/djapp/5_canary/metal_v2.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metal-v2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: metal
+      version: v2
+  template:
+    metadata:
+      labels:
+        app: metal
+        version: v2
+    spec:
+      containers:
+        - name: metal
+          image: "672518094988.dkr.ecr.us-west-2.amazonaws.com/hello-world:v1.0"
```

Comment:
Are the images mentioned in this example publicly available from this repository? Do customers need to build and push their own versions of these?

Reply:
The images are hosted in a dedicated account.

Reply:
@bcelenza https://github.com/aws/aws-app-mesh-examples/pull/129 adds the README and removes the .gitignore.